tests/test_core.py

```diff
@@ -9206,6 +9206,14 @@ def test_js_library_i64_params(self):
     self.emcc_args += ['--js-library=' + test_file('core/js_library_i64_params.js')]
     self.do_core_test('js_library_i64_params.c')
 
+  def test_attribute_destructor(self):
+    output = self.do_runf(test_file('core/test_attribute_destructor.c'), 'main done')
+    self.assertNotContained('my_dtor', output)
+
+    # Same again but with EXIT_RUNTIME
+    self.set_setting('EXIT_RUNTIME')
+    self.do_runf(test_file('core/test_attribute_destructor.c'), 'main done\nmy_dtor\n')
+
```

Comment:
How about moving this to `other`? That would avoid running the test in non-LTO modes where we don't need it. Also I don't think we run the lto tests on github CI here (just on chromium) so we'd get better coverage here on github.

Reply:
Good idea